src/styles/index.scss

```diff
@@ -1,3 +1,74 @@
 body {
+  display: grid;
+  place-items: center;
+  height: 100vh;
   margin: 0;
+  padding: 0;
+}
+
+.stopwatch {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 80vmin;
+  width: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+}
+
+.stopwatch__hand-minutes {
+  position: absolute;
+  top: 20vmin;
+  height: 20vmin;
+  width: 3vmin;
+  background-color: #0700ff;
+
+  transform-origin: bottom;
+  animation: minutes 3600s steps(60) infinite;
+}
+
+@keyframes minutes {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+.stopwatch__hand-seconds {
+  position: absolute;
+  top: 2vmin;
+  height: 38vmin;
+  width: 1.5vmin;
+  background-color: #2c8000;
+  transform-origin: bottom;
+
+  animation: seconds 60s linear infinite;
+}
+
+@keyframes seconds {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
```

Comment:
Don't need to duplicate keyframes with the same scenario, use only one keyframes